packages/ui/src/components/vuestic-sass/global/typography.demo.vue

```diff
@@ -1,34 +1,42 @@
 <template>
   <VbDemo>
-    <VbCard>
+    <VbCard title="Links">
       <a
         href="/default-link"
         class="link"
       >Default Link</a>
-    </VbCard>
-    <VbCard>
+      <br>
       <a
         href="/not-visited-link"
         class="link-secondary"
       >Secondary Link</a>
-    </VbCard>
-    <VbCard>
+      <br>
       <a
         href="#"
         class="link"
       >Default link Visited</a>
-    </VbCard>
-    <VbCard>
+      <br>
       <a
         href="#"
         class="link-secondary"
       >Secondary link Visited</a>
     </VbCard>
+    <VbCard title="Text styles">
+      <div>Default Text</div>
+      <div v-for="color in textStyleColors" :key="color" :class="`text--${color}`">{{ color }} text</div>
```

Comment:
According to https://github.com/epicmaxco/vuestic-ui/issues/876, we should use the `va` prefix in every of our classes. Please, make sure that the feature will not conflict with other frameworks.